src/link_creation_agent/equivalence_processor.cc

```diff
@@ -0,0 +1,133 @@
+#include "equivalence_processor.h"
+
+#include "Logger.h"
+#include "link_creation_db_helper.h"
+
+using namespace std;
+using namespace query_engine;
+using namespace link_creation_agent;
+using namespace atomdb;
+
+EquivalenceProcessor::EquivalenceProcessor() {}
+
+vector<string> EquivalenceProcessor::get_pattern_query(const vector<string>& c1,
+                                                       const vector<string>& c2) {
+    // clang-format off
+    vector<vector<string>> templates = {
+        {"OR", "2",
+            "LINK_TEMPLATE", "Expression", "3",
+                "NODE", "Symbol", "EVALUATION",
+                "LINK_TEMPLATE", "Expression", "2",
+                    "NODE", "Symbol", "PREDICATE",
+                    "VARIABLE", "P",
+                "LINK", "Expression", "2",
+                    "NODE", "Symbol", "CONCEPT"},
+                    c1,
+            {"LINK_TEMPLATE", "Expression", "3",
+                "NODE", "Symbol", "EVALUATION",
+                "LINK_TEMPLATE", "Expression", "2",
+                    "NODE", "Symbol", "PREDICATE",
+                    "VARIABLE", "P",
+                "LINK", "Expression", "2",
+                    "NODE", "Symbol", "CONCEPT"},
+                    c2
+        };
```

Comment:
The brackets don't seem to match properly. Please review. It seems like the opening in line 17, for instance, is closing in line 24.

Reply:
Ah, now I see the first `}` in line 16. So it seems like the brackets match BUT,  nonetheless, you should fix the `{` and `}` placement to make it clearer.